File: _CÓDIGO FONTE/Capítulo 5/04/hash.js
function minha_funcao_hash(login, senha) {
    let soma = 0;
    for (let caracter of login + senha) {
        soma += caracter.charCodeAt(0);
    }
    return soma;
}
    
console.log(minha_funcao_hash("lucas", "longo"));
console.log(minha_funcao_hash("lucas", "nova senha"));

File: _CÓDIGO FONTE/Capítulo 5/04/hash.py
def minha_funcao_hash(login, senha):
    
    soma = 0
    for caracter in login + senha:
        soma += ord(caracter)

    return soma

print(minha_funcao_hash("lucas", "longo"))
print(minha_funcao_hash("lucas", "nova senha"))
print(minha_funcao_hash("lucas", "nova senha"))

